### PR TITLE
Minor fixes for pulsar

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: sessions/17-heterogenous/ex1-pulsar.md [hunk 1]
```diff
@@ -264,10 +262,10 @@ $ sudo supervisorctl update
 pulsar: added process group
 ```
 
-Verify that Pulsar has started with Supervisor's log `tail` function:
+Verify that Pulsar has started by looking at pulsar's log file:
 
 ```console
-$ sudo supervisorctl tail pulsar
+$ more /srv/pulsar/var/uwsgi.log
```
Comment (nsoranzo) [inline sessions/17-heterogenous/ex1-pulsar.md]: s/more/less/ maybe?

File: sessions/17-heterogenous/ex1-pulsar.md [hunk 2]
```diff
@@ -403,7 +401,7 @@ Finally, we need to map a tool to the Pulsar destination. Do so by commenting ou
 Now, restart your Galaxy server:
 
 ```console
-$ sudo supervisorctl restart gx:*
+$ sudo supervisorctl restart galaxy:
```
Comment (nsoranzo) [inline sessions/17-heterogenous/ex1-pulsar.md]: That's not coherent with the following output, I think.

**Comment by mvdbeek** [comment]
You're right, I'll update that.

File: sessions/17-heterogenous/ex1-pulsar.md [hunk 3]
```diff
@@ -403,18 +401,22 @@ Finally, we need to map a tool to the Pulsar destination. Do so by commenting ou
...
 
 ```console
-$ sudo supervisorctl restart gx:*
-gx:handler0: stopped
-gx:handler1: stopped
-gx:galaxy: stopped
-gx:galaxy: started
-gx:handler0: started
-gx:handler1: started
+$ sudo supervisorctl restart galaxy:
```
Comment (nsoranzo) [inline sessions/17-heterogenous/ex1-pulsar.md]: I think a `*` is needed at the end of this line, but I'm not able to verify right now.

Comment (bgruening) [reply sessions/17-heterogenous/ex1-pulsar.md]: A star is not needed `:` just works fine.

Comment (nsoranzo) [reply sessions/17-heterogenous/ex1-pulsar.md]: Cool, good to know.

**Pull request merged**